DISCLAIMER.md

```diff
@@ -0,0 +1,27 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Copyrights & Trademarks](#copyrights--trademarks)
+* [Guidelines](#guidelines)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyrights & Trademarks
+
+The Simple Icons library contains a number of icons with specific requirements around their usage. Where this is the case, we aim to provide access to the proper guidelines through the `guidelines` entry in our JSON file & in our NPM package and through links on our [website](http://simpleicons.org/).
+
+We ask that users of our library read these guidelines fully and display the icons in accordance with them and, where applicable, with the necessary permissions.
```

Comment:
The word of the use "library" seems (technically speaking) a bit too restrictive, I think the easiest way to generalize this sentence is as follows. But I'm open to other suggestions

```suggestion
We ask that users of icons in the collection read these guidelines fully and display the icons in accordance with them and, where applicable, with the necessary permissions.
```